dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
+
+public class Demo {
+    public static final String RPOSITORY = "testRepo";
+    public static void main(String[] args) {
+        int wek = 4;
+        String day;
+    
+        switch (wek) {
```

Comment:
Possible typo in `wek`, fixes:
Candidate #1 with a confidence 37%
```suggestion
        switch (week) {
```
Candidate #2 with a confidence 31%
```suggestion
        switch (weak) {
```
Candidate #3 with a confidence 31%
```suggestion
        switch (web) {
```

<!-- lookout footnote separator -->
_Comment made by the analyzer style-check._